apps/artemis-web/components/footer/footer.spec.tsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
-import { render } from '@testing-library/react';
-
+import Enzyme, { mount } from 'enzyme';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
 import Footer from './footer';
 
+Enzyme.configure({ adapter: new Adapter() });
+
 describe('Footer', () => {
   it('should render successfully', () => {
-    const { baseElement } = render(<Footer />);
-    expect(baseElement).toBeTruthy();
+    const element = mount(<Footer system_version={'2.0.0'} />);
```

Comment:
this should be dynamic from .env, not hardcoded (same for HEAD, it should point to the git commit id)

Reply:
Why? The purpose of this is to test that the version is rendered properly. We don't really care about the value...

Reply:
I see. Could you point me to the code where we dynamically render the version and git revision? I get that this point here is about correct rendering, I would just like to understand if we do any erroneous hardcoding in practice (which then makes extremely difficult for the users to let us know what actual code version they run on-premise)

Reply:
https://github.com/FORTH-ICS-INSPIRE/artemis-web/blob/nextjs_migration/apps/artemis-web/components/footer/footer.tsx